docs/reference/cli/using-the-cli.md

```diff
@@ -92,6 +92,29 @@ Skip to the next track on player `00:04:20:ab:cd:ef`:
     ```sh
     printf "00:04:20:ab:cd:ef playlist index +1\n" | nc 192.168.1.1 9090
     ```
+    
+### Practice with a few Telnet commands
+Log in to Telnet using port 9090. Use Telnet commands to get version of LMS, find player id, and use that player id to mute the player. After each command you should get a response.

```

Comment:
I'd call them "CLI commands" rather than "Telnet commands", as one might think they could checkout the telnet documentation to get help.